### ISSUE TITLE
Duplicating transformations does not appear to be working

### ISSUE THREAD

**Issue by javaworms** [fiji/SPIM_Registration#93]
I am using the mCherry channel for interest point detection and registration, and would like to apply the same transformations for the mCherry channel to the GFP channel for each timepoint. I go to Batch Processing -> Tools -> Duplicate Transformations, select "One channel to other channels", apply to "all angles", "all illuminations", "single timepoint" (just using a single timepoint for testing purposes). In the next window, I select mCherry as the source channel, GFP as the processing channel. 

I get the following on the log:

Using spimdata version: 0.9-revision
Using spimreconstruction version: 4.0.0
angles selected: 0, -90
channels selected: GFP, mCherry
illuminations selected: 0
Timepoints selected: 429
Applying chanel mCherry >>> GFP
Source viewId t=429, ch=mCherry, ill=0, angle=0
Target viewId t=429, ch=GFP, ill=0, angle=0
Adding model Redundant geometric local descriptor matching (translation invariant), AffineModel3D regularized with an RigidModel3D, lambda = 0.1 on [nuclei (c=mCherry)], 3d-affine: (1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0)
Source viewId t=429, ch=mCherry, ill=0, angle=-90
Target viewId t=429, ch=GFP, ill=0, angle=-90
Adding model Redundant geometric local descriptor matching (translation invariant), AffineModel3D regularized with an RigidModel3D, lambda = 0.1 on [nuclei (c=mCherry)], 3d-affine: (0.9717305959411154, 0.06476190985089271, 0.01785583251652962, 129.0735636361079, -0.025852503745879503, 0.999668800319163, 0.02503920289641915, 6.6269721298016115, 0.001449382296780424, 6.361026366212074E-4, 0.9936427939073234, 162.38830922995416)

However, this does not appear to do anything. The only way I can get the GFP channel to match up with the mCherry channel is to open the registration explorer and manual copying the "redundant geometic local descriptor" row from one channel to the other. And even then, the new view does not appear in bigdataviewer until I close and reopen the program. 

Any help would be much appreciated.

**Comment by StephanPreibisch** [comment]
Hi, while doing that the MultiView Reconstruction app needs to closed (after saving the XML) as the duplicate transformation at the moment simply re-saves the XML. Did you already do that?

**Comment by peteyk** [comment]
Hi, you were right, I needed to immediately close the program. When I was doing duplicate transformations on all timepoints before, I didn't see any changes, so I proceeded to try other operations, and after closing and reopening the duplicate changes still didn't show up. It looks like I have to close the program before trying anything else to get the changes to stick. 

It's not the biggest deal right now, but someday it might be nice not to have to close and reopen the app to see the latest transformation changes.

Thanks for your help!
